workers/generic-worker/tclog/tclog.go

```diff
@@ -0,0 +1,21 @@
+package tclog
+
+type Logger interface {
+	Log(prefix, message string)
+	Info(message string)
+	Infof(format string, v ...interface{})
+	Warn(message string)
+	Warnf(format string, v ...interface{})
+	Error(message string)
+	Errorf(format string, v ...interface{})
+}
```

Comment:
It looks like this is only used for `Artifacts.GetLatest`?  Is the plan to use it more extensively?  Some commentary on what it's for and what is expected to satisfy the interface (I would not have guessed a task!) would be helpful to the reader and anyone using or modifying this later.

Reply:
That was a bit of a mess. There was a utility method to download from a URL which was used by the mounts feature and also by the mock framework. I had pulled it out, but not finished documenting it or cleaning it up. In a cleanup since, I've removed the artifact interface that I previously had, that was using it, so now it is only used by mounts feature, and since it has mounts feature specific logging in, I've removed the tclog package entirely, it isn't needed.